go.mod

```diff
@@ -144,3 +144,5 @@ require (
 	gopkg.in/ini.v1 v1.66.4 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace golang.org/x/crypto v0.0.0-20220313003712-b769efc7c000 => github.com/pmm-experiments/crypto v0.0.0-20220330205032-9c28ba91f872
```

Comment:
@tshcherban can we move all repos from pmm-experiments into `Percona-Lab` organization we already have?

Reply:
Sure, its just for development since I don't have write access to `Percona-Lab` org

Reply:
Ok, I'll ask to give you a permission.